library.properties

```
@@ -0,0 +1,9 @@
+name=ESP32_Button
+version=1.0.0
+author=Ali Hassan shah <ali.shah@espressif.com>
```

```suggestion
author=espressif
```
This filed will be shown in Arduino IDE. So using the company's name is better.